.github/workflows/ci.yml

```diff
@@ -175,9 +175,9 @@ jobs:
             ~/.cargo/registry
             ./target
           key:
-            ${{ matrix.os }}-${{ matrix.kind }}-${{ hashFiles('Cargo.lock') }}
+            a-${{ matrix.os }}-${{ matrix.kind }}-${{ hashFiles('Cargo.lock') }}
```

Comment:
This is needed because rusty_v8 doesn't automatically download a new version of librusty_v8.a.